22-decay-tree-fitter.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: HowTo DecayTreeFitter
+subtitle: How do I use DecayTreeFitter? 
+minutes: 10
+---
+
+> ## Learning Objectives {.objectives}
+>
+> * Add a kinematic fitter to a branch in the decay tree
+> * Apply a mass constraint
+> * Inspect the refitted decay tree
+
+Once you have made a hypothesis on the chain of decays that have lead to your final state, you then can incorporate the additional knowledge that comes with this hypothesis to get a new best estimate for the particle parameters - in particular their momenta. The additional knowledge is represented as constaints, which your decay tree has to fulfill.
+
+For example for the decay
+```python
+"[D*(2010)+ -> (D0 -> K- pi+) pi+]CC"
+``` 
+you make the assumption that the (K- pi+) combine to form a D0 with a specific invariant mass. This results in a so called *mass-constraint*. In adition the Kaon and the pion are supposed to originate from exactly the same point in space and the D* should come from the primary vertex. Boundary conditions like those are called *vertex-constraints*. 
```

Comment:
“adition” → “addition”. “Kaon” → “kaon”. 

The D* should not _necessarily_ come from the PV, of course, it can be produced in B decays and the like. It might be worth being pedantic here as it's a physics issue (i.e. you shouldn't _always_ use DTF with this decay).